hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisServer.java

```diff
@@ -126,15 +126,32 @@ private static long nextCallId() {
   public OMResponse submitRequest(OMRequest omRequest) throws ServiceException {
     RaftClientRequest raftClientRequest =
         createWriteRaftClientRequest(omRequest);
```

Comment:
`createWriteRaftClientRequest` was creating a request with the same callId. Is that the bug here?
If that is the case we would also need to make changes in this function so that callId is different for every request.

Reply:
When called from Client, Server.getCallID() will not return constant value RpcConstants.INVALID_RETRY_COUNT.
Where as when keydeletingService is called it is not coming thru RpcClient, directly calling server API submitRequest, so CallID will be same for each call.

So it is fine use createWriteRaftClientRequest for normal flow of requests, and the values ClientID and CallID can be fetched from Server.getClientID and Server.getCallID().

Reply:
Can we add an assertion to check that Server.getCallID() is not returning RpcConstants.INVALID_CALL_ID and Server.getClientId() is not returning RpcConstants.DUMMY_CLIENT_ID?

Reply:
Done.